### DISCUSSION TITLE
Too many visual glitches with start Menu, M$ Outlook and Toast notifications

### DISCUSSION THREAD

**Discussion by audioscavenger** [rustdesk/rustdesk#6581] [General]
### Bug Description

* cannot interact with Outlook toast notifications
* Outlook cannot show calendar
* scrolling is broken in folders and calendar
* Outlook file menu won't even show properly
* Windows start menu won;t show properly

![image](https://github.com/rustdesk/rustdesk/assets/3853962/5fa2eb7b-7cf3-4d61-aacf-312cc8582e8a)

![image](https://github.com/rustdesk/rustdesk/assets/3853962/696e857c-ff05-466d-92cd-076de9404bce)

### How to Reproduce

* both server and client upgraded to latest
* using self docker server on ARM-64
* connect via LAN

### Expected Behavior

no glitches

### Operating system(s) on local side and remote side

Windows 10 --> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/3853962/5fa2eb7b-7cf3-4d61-aacf-312cc8582e8a)

![image](https://github.com/rustdesk/rustdesk/assets/3853962/696e857c-ff05-466d-92cd-076de9404bce)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/6551
